src/Utils/IconLibrary.tsx

```diff
@@ -52,6 +52,7 @@ export const ICONS = {
   FILE: <path d="M82.04 21.485c2.065 2.065 3.795 6.195 3.795 9.152v64.286c0 2.957-2.4 5.357-5.357 5.357h-75A5.359 5.359 0 0 1 .12 94.923V5.637C.12 2.68 2.52.28 5.478.28h50c2.957 0 7.087 1.73 9.151 3.795l17.411 17.41zM57.263 7.87v20.982h20.982c-.334-.948-.837-1.897-1.227-2.288L59.55 9.097c-.39-.39-1.339-.893-2.288-1.228zm21.429 85.268V35.994H55.478a5.359 5.359 0 0 1-5.358-5.357V7.423H7.263v85.714h71.429zM21.549 44.923c0-1.005.781-1.786 1.786-1.786H62.62c1.005 0 1.786.781 1.786 1.786v3.571a1.763 1.763 0 0 1-1.786 1.786H23.335a1.763 1.763 0 0 1-1.786-1.786v-3.571zm41.071 12.5c1.005 0 1.786.781 1.786 1.786v3.571a1.763 1.763 0 0 1-1.786 1.786H23.335a1.763 1.763 0 0 1-1.786-1.786v-3.571c0-1.005.781-1.786 1.786-1.786H62.62zm0 14.286c1.005 0 1.786.78 1.786 1.785v3.572a1.763 1.763 0 0 1-1.786 1.785H23.335a1.763 1.763 0 0 1-1.786-1.785v-3.572c0-1.004.781-1.785 1.786-1.785H62.62z" />,
   FILE_ALTERNATE: <path d="M 85.667969 20.582031 L 66.917969 1.832031 C 65.746094 0.660156 64.15625 0 62.5 0 L 18.75 0 C 15.296875 0 12.5 2.796875 12.5 6.25 L 12.5 93.75 C 12.5 97.203125 15.296875 100 18.75 100 L 81.25 100 C 84.703125 100 87.5 97.203125 87.5 93.75 L 87.5 25 C 87.5 23.34375 86.839844 21.753906 85.667969 20.582031 Z M 50 75 L 31.25 75 L 31.25 62.5 L 50 62.5 Z M 68.75 56.25 L 31.25 56.25 L 31.25 43.75 L 68.75 43.75 Z M 68.75 25 C 65.296875 25 62.5 22.203125 62.5 18.75 L 62.5 6.25 L 81.25 25 Z M 68.75 25 " />,
   FILE_SOLID: <path d="M54.072 34.976h27.765L54.072 7.452v27.524zM19.096 0H59.24l30.048 30.048v60.096c0 2.725-1.001 5.048-3.004 6.971C84.28 99.038 81.917 100 79.192 100H19.096c-2.724 0-5.088-.962-7.091-2.885C10.002 95.192 9 92.87 9 90.145l.24-80.29c0-2.724.962-5.047 2.885-6.97C14.048.962 16.372 0 19.096 0z" />,
+  FILTER: <path d="M75,70 C78.3137085,70 81,72.6862915 81,76 L81,79.357 L100,79.3571429 L100,90.5 L81,90.499 L81,94 C81,97.3137085 78.3137085,100 75,100 L68,100 C64.6862915,100 62,97.3137085 62,94 L62,90.499 L0,90.5 L0,79.3571429 L62,79.357 L62,76 C62,72.6862915 64.6862915,70 68,70 L75,70 Z M32,35 C35.3137085,35 38,37.6862915 38,41 L38,44.357 L100,44.3571429 L100,55.5 L38,55.499 L38,59 C38,62.3137085 35.3137085,65 32,65 L25,65 C21.6862915,65 19,62.3137085 19,59 L19,55.499 L0,55.5 L0,44.3571429 L19,44.357 L19,41 C19,37.6862915 21.6862915,35 25,35 L32,35 Z M75,0 C78.3137085,0 81,2.6862915 81,6 L81,9.357 L100,9.35714286 L100,20.5 L81,20.5 L81,24 C81,27.3137085 78.3137085,30 75,30 L68,30 C64.6862915,30 62,27.3137085 62,24 L62,20.5 L0,20.5 L0,9.35714286 L62,9.357 L62,6 C62,2.6862915 64.6862915,0 68,0 L75,0 Z" id="path-1" />,
```

Comment:
Oh @rosslo you should be able to remove the `id="path-1"` attribtue

Reply:
Updated!